account/signals.py

```diff
@@ -12,3 +12,4 @@
 email_confirmed = django.dispatch.Signal(providing_args=["email_address"])
 email_confirmation_sent = django.dispatch.Signal(providing_args=["confirmation"])
 password_changed = django.dispatch.Signal(providing_args=["user"])
+updated_account = django.dispatch.Signal(providing_args=["user", "request"])
```

Comment:
This should follow the naming pattern of the other signals. `<noun>_<past-tense-verb>`.

Reply:
```suggestion
account_updated = django.dispatch.Signal(providing_args=["user", "request"])
```